networkx/linalg/algebraicconnectivity.py

```diff
@@ -377,6 +377,21 @@ def algebraic_connectivity(
     See Also
     --------
     laplacian_matrix
+
+    Examples
+    --------
+    For undirected graphs algebraic connectivity can tell us if a graph is connected or not
+    G is connected iff  :math: `algebraic\_connectivity(G)>0`
```

Comment:
Just a nit here but I think this is more readable (in the docstring at least) as an inline literal rather than trying to format it as math:

```suggestion
    `G` is connected iff  ``algebraic_connectivity(G) > 0``:
```